docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,194 @@
+
+In the following example, the username and password fields for ZOSMF1 and ZOSMF2 are empty to allow them to inherit values from the base array:
+
+<!-- Add JSON example here where 2 zosmf services are inheriting user and pass from base array -->
+
+### Tips for using the base array
+
+The base array is a useful tool for sharing option values between services. You might define options to the base array in the following situations:
+- You have multiple services that share the same username, password, or other value.
+- You want to store a web token to access all services through Zowe API Mediation Layer.
+- You want to trust a known self-signed certificate, or your site does not have server certificates configured. You can define `reject-unauthorized` in the base array with a value of  `false` to apply to all services. Understand the security implications of accepting self-signed certificates at your site before you use this method.
+
+<!-- Any tips or examples you can think of that could be helpful here in this "Managing configuration efficiently" section, besides the base array? -->
+## Managing credential security
+
+<!-- This section is in progress, just need to turn the notes into better writing. -->
+
+When you first run the `cnfg init --global` command, you get config where your username and password are set to secure.
+
+You can define other fields in the secure array manually as well to secure them. After initializing, the profiles.base.properties.user and profiles.base.properties.password fields are defined to the secure array in global zowe.config.json. Any option that you define to secure array will become secure/prompted for.
+
+The `zowe cnfg secure` command can re-prompt for all secure fields when you want to update them (such as password change)
+
+The command `zowe cnfg set secure --password` prompts you specifically for password only (substitute whatever option name you want instead of password)
+
+ ## Sharing global configuration
+
+<!-- How to push global config to a code repository, and how to consume config as a member of a project.
+
+I don't know much about this. I assume you'd need to post your config file somewhere in github (or even as simple as sending it in an email) and that other person needs to grab it and place it in their .zowe/settings folder. Then issue commands to test. -->
+
+## Overriding global configuration
+
+<!--
+You can edit your user config to explicitly do something different from what's defined in global.
+
+You don't really need to use this. But you can use this if you don't plan to share, or want to override things others have shared.
+
+If you don't use it, it's just a copy of your init config file.
+-->
+
+## Example configurations
+
+<!--
+Shall we provide a handful of examples here of different use cases and the .json for each? The article covered the basics, how much will people be able to extrapolate from there?
```

Comment:
I'd cover at least two. One for accessing multiple services directly on multiple LPARs that share the same username/password. One for accessing multiple services via the API ML (where MFA/SSO is achievable via token-based auth). 
Another example could be accessing multiple services directly on LPAR1 and LPAR2 where username/password varies between LPAR1 & LPAR2 services.
Yet another example could be a situation where API ML is leveraged to access production services but services on a dev-test environment can be accessed directly.